lightning/src/ln/channelmanager.rs

```diff
@@ -484,6 +483,12 @@ pub struct ChannelManager<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref,
 	/// PersistenceNotifier the lock contains sends out a notification when the lock is released.
 	total_consistency_lock: RwLock<()>,
 
+	/// We want to (optionally) send fee updates on our outbound channels at least once after after
```

Comment:
What do you mean here by optionally ? Either the use can opt-in/opt-out of this behavior or the update fee is conditional to a feerate fluctuation reported by fee-estimator ?